docs/architecture/grpc-for-wcf-developers/self-hosted.md

```diff
@@ -139,21 +144,24 @@ sudo journalctl -u myapp
 > [!TIP]
 > If you have a GUI environment available on your host, a few graphical log viewers are available for Linux, such as *QJournalctl* and *gnome-logs*.
 
-To learn more about querying the systemd journal from the command line with `journalctl`, see [the man pages](https://manpages.debian.org/buster/systemd/journalctl.1).
+To learn more about querying the `systemd` journal from the command line by using `journalctl`, see [the man pages](https://manpages.debian.org/buster/systemd/journalctl.1).
+
+## HTTPS certificates for self-hosted applications
 
-## HTTPS Certificates for self-hosted applications
+When you're running a gRPC application in production, you should use a TLS certificate from a trusted certificate authority (CA). This CA might be a public CA, or an internal one for your organization.
 
-When running a gRPC application in production, you should use a TLS certificate from a trusted Certificate Authority (CA). This CA could be a public CA, or an internal one for your organization.
+On Windows hosts, you can load the certificate from a secure [certificate store](https://docs.microsoft.com/windows/win32/seccrypto/managing-certificates-with-certificate-stores) by using the [X509Store class](https://docs.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.x509store?view=netcore-3.0). You can also use the `X509Store` class with the OpenSSL key store on some Linux hosts.
 
-On Windows hosts, the certificate may be loaded from a secure [Certificate Store](https://docs.microsoft.com/windows/win32/seccrypto/managing-certificates-with-certificate-stores) using the [X509Store class](https://docs.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.x509store?view=netcore-3.0). The `X509Store` class can also be used with the OpenSSL key-store on some Linux hosts.
+You can also create certificates by using one of the [X509Certificate2 constructors](https://docs.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.x509certificate.-ctor?view=netcore-3.0), from either:
```

Comment:
@shirhatti something looks wrong here. The link text says X509Certificate2 and the link goes to X509Certificate. Which one is correct so I can properly link to the API page using the xref format.

Reply:
It should be `X509Certificate2`

Reply:
```suggestion
You can also create certificates by using one of the [X509Certificate2 constructors](xref:System.Security.Cryptography.X509Certificates.X509Certificate2.%23ctor%2A), from either:
```